graphql-server/pkg/queries/test.go

```diff
@@ -29,3 +30,16 @@ func (q *TestQuery) QueryTestByID(id string) (*models.Test, error) {
 
 	return test, nil
 }
+
+func (q *TestQuery) QueryTestsByIDs(ids []string) ([]*models.Test, []error) {
+	m := make([]*models.Test, 0, len(ids))
+
+	// TODO: figure out how to handle this with CTE
+	// https://bun.uptrace.dev/guide/query-where.html#where-values
+	err := q.session.NewSelect().Model(&m).Where("id IN (?)", bun.In(ids)).Scan(q.ctx)
+	if err != nil {
+		return nil, []error{err}
+	}
+
```

Comment:
yes I tried to do it using CTE provided in the [docs](https://bun.uptrace.dev/guide/query-where.html#where-values) but no luck so i ended up leaving a TODO first

```
err := db.NewSelect().
	With("data", db.NewValues(&users).WithOrder()).
	Model(&users).
	Where("user.id = data.id").
	OrderExpr("data._order").
	Scan(ctx)
```

Reply:
ok... but the current code will not work as the dataloader library expects result ordering?

how about handling on golang side first? [ref](https://github.com/oursky/project-house/blob/ec8d87c9e9fb735fdb71b8aaf23b7d253aa518a1/server/pkg/query/user.go#L35)

Reply:
alternatively, can we try `WherePK`?

Reply:
`WherePK` produces a `WHERE IN` so we will need to manage the order ourselves i think

Reply:
let's throw in loads of for loops on the golang side first

Reply:
WherePK:

```
var tests []*model.Test
for _, id := range ids {
    tests = append(tests, &model.Test{Id: id})
}
err := db.NewSelect().Model(&tests).WherePK("id").Scan(ctx)
```

this will fill the result array in the order we want?

Reply:
o, can try

Reply:
seems the order is not retained, let's do it the traditional way